debian/update-dch-from-git

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I tend to think, that this kind of operation should be done directly in the Makefile.